crates/utils/src/utils/validation.rs

```diff
@@ -302,12 +302,22 @@ pub fn check_site_visibility_valid(
   Ok(())
 }
 
+pub fn check_url_scheme(url: &Option<Url>) -> LemmyResult<()> {
+  if let Some(url) = url {
+    if url.scheme() != "http" && url.scheme() != "https" {
```

Comment:
`["https", "http"].iter().any(|s| s == &url.scheme())`

Reply:
This is too confusing for such a simple check.